.node-version

```diff
@@ -0,0 +1 @@
+16
```

Comment:
shall we put an engine version in the package json too?

Reply:
Yeah, it can be. I just pushed this change to my branch.